src/deadline/client/ui/widgets/host_requirements_tab.py

```diff
@@ -843,6 +1010,18 @@ def add_requirement(self, requirements: List, name: str, scaling_factor: int = 1
                 requirement["max"] = self.max_spin_box.value() * scaling_factor
             requirements.append(requirement)
 
+    def set_minimum(self, minimum_value: int, scaling_factor: int = 1):
+        if minimum_value == self.min_spin_box.no_input_value:
+            self.min_spin_box.setValue(self.min_spin_box.no_input_value)
+        else:
+            self.min_spin_box.setValue(minimum_value // scaling_factor)
+
+    def set_maximum(self, maximum_value: int, scaling_factor: int = 1):
+        if maximum_value == self.max_spin_box.no_input_value:
+            self.max_spin_box.setValue(self.max_spin_box.no_input_value)
+        else:
+            self.max_spin_box.setValue(maximum_value // scaling_factor)
+
 
```

Comment:
Could we use `@setter` here?

Reply:
I think a setter here could be possible, but we'd have to do the scaling factor work outside of the setter. How do we feel about that?

Reply:
I think that's fine. If the calling method is already defining the scaling factor itself, it shouldn't be an issue to do the scaling there instead- using `@setter` is also better for consistency